.gitignore

```diff
@@ -3,7 +3,6 @@ vouch-proxy
 main
 config/config.yml
 config/*config.yml
-config/config.yml_*
```

Comment:
This is a convenience for my personal workflow in my dev enviroment.  I use symbolic links to switch through various configs and IDPs.  I'm gonna keep it mostly because I want it.  If there's a real problem there, please do point it out.